examples/README.md

```diff
@@ -0,0 +1,109 @@
+The csp-billing-adapter requires a configuration file, (defaults to `~/.config/csp_billing_adapter/config.yaml`)
+in order to run successfully. The configuration files in in the `examples` directory 
+contain examples for the different product configurations. 
+
+This README provides a listing of all of the fields and descriptions.
+Fields that are optional are labeled accordingly.
+
+```
+version:
+    The version of the adapter implementation.
+
+billing_interval:
+    Sets the billing interval time period.
+
+api: (OPTIONAL)
+    Provides the path to the application information the csp-billing-adapter reports to the CSP API.
+    This is shared with the application the csp-billng-adapter is reporting about. The application may provide this as an endpoint directly, i.e. http://localhost/$API works.
+
+product_code:
+    TBD
+
+query_interval:
+    Time unit how often the csp-billng-adapter collectd the data from the application.
+
+reporting_api_is_cumulative:
+    Specifies if the CSP will accumulate reported values. While all CSPs will accumulate the values reported, it may not be possible to report 0. For example, in a monthly billing scenario, the csp-billing-adapter may have to report on a hourly basis a number that is not 0 such that the CSP API accumulates the values. A true setting would direct the adapater to normalize the unit count based on some to be determined normalization algorithm.
+
+reporting_interval: 
+    Sets the time when csp-billing-adapter reports to the CSP API. The values the csp-billing-adapter reports are determined by a combination of the settings for `reporting_api_is_cumulative` and `billing_interval`.
+
+support_info_collection_cmd:
```

Comment:
I thought this was being dropped, but in the structs agreed upon in the April 13 meeting we have one config with this setting in the config and one without. It should be consistent. @smarlowucf you stated theis would be hardcoded in a CSP specific plugin?

Reply:
Yes, there's an issue for this https://github.com/SUSE-Enceladus/csp-billing-adapter/issues/20. I was not sure if we still want the option. It's not needed if info is part of csp plugins.

Reply:
so keep or remove?

Reply:
seems fine to remove

Reply:
fixed in https://github.com/SUSE-Enceladus/csp-billing-adapter/pull/38